.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "root": true,
+  "extends": ["./node_modules/sanctuary-style/eslint-es6.json"],
```

Comment:
Weren't you going to override the directive that matches the files to only match `.mjs`?

Reply:
I would need to create an override section for the `index.js` file. That does not seems like a neat solution and I would prefer having a `.eslintignore`. But as the `index.js` is auto generated and it shouldn't be edited manually I think adding an exception is not needed.